doc/rst/developer/bestPractices/TestSystem.rst

```diff
@@ -285,15 +295,22 @@ single-locale setting:
      CHPL_COMM != none
 
 This is useful when the conditions required to skip a test can be easily
-determined from the environment.
+determined from the environment.  A condition of ``>=`` indicates that the test
+should be skipped when the environment variable on the left does not contain the
+contents on the right, while ``<=`` indicates the opposite - this is useful for
+imprecise matches, e.g. ``CHPL_HOST_PLATFORM >= cygwin`` would cause the test to
+run on both ``cygwin64`` and ``cygwin32``.
```

Comment:
I think this would be a bit easier to follow if ``<=`` were given the explicit definition, since it has fewer "not"s.